test/utils/schedulingUtility_test.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:cie_team1/model/course/course.dart';
+import 'package:cie_team1/utils/schedulingUtility.dart';
+import 'package:cie_team1/utils/staticVariables.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+
+@Timeout(const Duration(seconds: 5))
+void main() {
+
+
+  group('far campus', ()
+  {
+
+
```

Comment:
Would be great to hit 'Ctrl + Alt + L' for code formatting. This should normally remove all those empty lines